doc/usermanual.md

```diff
@@ -1,218 +1,238 @@
-# User & Programmers Manual
-
-* [Usage](#usage)
-  + [Stats Registry](#stats-registry)
-  + [Alarm module](#alarm-module)
-  + [Logs](#logs)
-  + [Transactions](#transactions)
-  + [Library overview](#library-overview)
-  + [Function reference](#function-reference)
-* [Development documentation](#development-documentation)
-  + [Contributions](#contributions)
-  + [Project build](#project-build)
-  + [Testing](#testing)
-  + [Coding guidelines](#coding-guidelines)
-  + [Continuous testing](#continuous-testing)
-  + [Code Coverage](#code-coverage)
-  + [Clean](#clean)
-
+# Library Functions
+
+-   [Usage](#usage)
+    -   [Stats Registry](#stats-registry)
+    -   [Alarm module](#alarm-module)
+    -   [Transactions](#transactions)
+    -   [Library overview](#library-overview)
+    -   [Function reference](#function-reference)
+-   [Development documentation](#development-documentation)
```

Comment:
Not sure but it seems this piece of docs has been moved to a separate document (usermanual.md), so the sub-ToC here wouldn't make too much sense...

Reply:
Fix 9844afe